Source/Scene/Scene.js

```diff
@@ -868,11 +868,9 @@ define([
                 break;
             }
 
-            if (command.pass === Pass.OPAQUE || command instanceof ClearCommand) {
-                frustumCommands.opaqueCommands[frustumCommands.opaqueIndex++] = command;
-            } else if (command.pass === Pass.TRANSLUCENT){
-                frustumCommands.translucentCommands[frustumCommands.translucentIndex++] = command;
-            }
+            var pass = command instanceof ClearCommand ? Pass.OPAQUE : command.pass;
+            var index = frustumCommands.indices[pass]++;
+            frustumCommands.commands[pass][index] = command;
```

Comment:
That is not ideal for performance, right?  Perhaps we should allocate (and reallocate as needed) the worse case based on the number of commands.  Doesn't need to be in this PR unless you feel like it.